soe_basic.info.yml

```diff
@@ -29,6 +29,7 @@ component-libraries:
 
 libraries:
   - stanford_basic/basic
```

Comment:
Do you need to declare the parent library here? Try removing it and see what happens.

Reply:
I also wondered that. I think it's old code leftover and we can remove it. It compiles fine without it.